banking-app/Makefile

```diff
@@ -10,23 +10,45 @@ build: ## 🔨 Build the Banking Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
 
-test: ## 🧪 Test the Banking Application
+test: start-host-for-test ## 🧪 Test the Banking Application in the sandbox
```

Comment:
How about creating test_local.sh instead of `start-host-for-test` and `stop_sandbox.sh`?
test_local.sh would be something like this (it's very rough, but hopefully you will get the idea):
```bash
<ccf>/sandbox.sh <options>
sandbox_pid=$!

function finish {
  kill -9 $sandbox_pid
}
trap finish EXIT

# waiting for sandbox etc...

./test.sh
```

It would be safer in terms of stopping correct process (sandbox).

Reply:
Great plan - what do you think now @takuro-sato?